ports/esp32/boards/ESP32_GENERIC_C3/mpconfigboard.h

```diff
@@ -4,7 +4,7 @@
 #define MICROPY_HW_MCU_NAME                 "ESP32C3"
 
 #define MICROPY_HW_ENABLE_SDCARD            (0)
-#define MICROPY_PY_MACHINE_I2S              (0)
+#define MICROPY_PY_MACHINE_I2S              (1)
```

Comment:
Please just remove this line altogether.  Then it should pick up the default config, which is to enable I2S.